viewing_party/party.py

```diff
@@ -0,0 +1,196 @@
+def create_movie(movie_title, genre, rating):
+
+    if movie_title and genre and rating:
+        return {"title" : movie_title, "genre" : genre, "rating" : rating}
+    return None
+   
+
+def add_to_watched(user_data, movie):
+  
+    user_data["watched"].append(movie)
+    return user_data
+   
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(janes_data, Title_A):
+
+    for each in janes_data["watchlist"]:
+
+         if Title_A in each.values():
+            janes_data["watched"].append(janes_data["watchlist"].pop())
+         
+
+    return janes_data
+    
+
+def get_watched_avg_rating(janes_data):
+    
+    sum_counter = 0.0
+    for each in janes_data["watched"]:
+           sum_counter += each["rating"]
+    if len(janes_data["watched"]):
+
+         return sum_counter/len(janes_data["watched"])
+    else:
+        return 0
+
+
+def get_most_watched_genre(janes_data):
+  genre_dict = {}
+    
+  for listi in janes_data["watched"]:
```

Comment:
Great work implementing this frequency map! As above, consider a more meaningful name for `listi`